### PR TITLE
Polisher and Making Mirror

### PR DESCRIPTION
**Pull request by Dozed12**
#11336
- Polisher - Done
- Spawn - Done ( may add more)
- Mirror recipie - Done

### CODE REVIEW

**Comment by KA101** [comment]
This kinda needs the recipe before it's mergeable.

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -5050,7 +5050,39 @@
...
+  "type" : "recipe",
+  "result": "mirror",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "fabrication",
+  "difficulty": 2,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
+  "tools": [
```
Comment (KA101) [inline data/json/recipes/recipe_others.json]: OK, if you're using a forge I'm looking for needing a HAMMER quality to shape the steel.

**Comment by KA101** [comment]
Clearing the hopper, will fix in the merge.

File: data/json/recipes/recipe_others.json [hunk 2]
```diff
@@ -5050,7 +5050,39 @@
       [ "silver_small", 50 ]
      ]
     ]
-},{
+},
+{
+  "type" : "recipe",
+  "result": "mirror",
```
Comment (KA101) [inline data/json/recipes/recipe_others.json]: Oops, something I neglected to mention.  There's already a recipe for "mirror", so this conflicts.  We add an "id_suffix" line to fix this; I'll go with "from_steel".

File: data/json/recipes/recipe_others.json [hunk 3]
```diff
@@ -5050,7 +5050,39 @@
...
+      ["crucible", -1]
+     ],
+     [
+      ["forge", 50], ["char_forge", 10]
+     ]
+    ],
+  "components":
+    [
+     [
+      ["chunk_steel", 3]
```
Comment (KA101) [inline data/json/recipes/recipe_others.json]: steel_chunk, actually.  I'm getting the impression that you Used The Web Interface (insert scare chord here).

**Comment by Dozed12** [comment]
I usually write it in Notepad and then paste it in the web developer for these small addictions. I knew there was another recipe, I didnt knew I had to differenciate them in some way.

**Comment by kevingranade** [comment]
You wouldn't need to know, the game would tell you when you tried to load
your new file, which you should ALWAYS do.

File: data/json/recipes/recipe_others.json [hunk 4]
```diff
@@ -5050,7 +5050,39 @@
...
+{
+  "type" : "recipe",
+  "result": "mirror",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "fabrication",
+  "difficulty": 2,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
```
Comment (BevapDin) [inline data/json/recipes/recipe_others.json]: This recipe is completely unavailable to the player, it is neither auto learned, nor learned through disassembling (non-reversible) nor is it in any book.

The other hand mirror recipe can be learned from the glass blowing book. It contains this:
```JSON
  "book_learn": [[ "glassblowing_book", 5 ]],
```
Maybe this should be added to this recipe as well?

Comment (KA101) [inline data/json/recipes/recipe_others.json]: Should've been autolearn, I'll fix that in the merge.

**Pull request merged**